compiam/conf/raga/carnatic.yaml

```diff
@@ -1,35 +1,295 @@
+56	Shanmukhapriya:	
+               arohana: ['S', 'R₂', 'G₂', 'M₂', 'P', 'D₁', 'N₂', 'Ṡ'] 
+               avarohana: [ 'S', 'N2', 'D1', 'P', 'M2', 'G2', 'R2', 'S']
+57	Simhendramadhyamam:	
+              arohana: ['S', 'R₂', 'G₂', 'M₂', 'P', 'D₁', 'N₃', 'Ṡ']
+              avarohana: ['S', 'N3', 'D1', 'P', 'M2', 'G2', 'R2', 'S']
+58	Hemavati:	
+              arohana: ['S', 'R₂', 'G₂', 'M₂', 'P', 'D₂', 'N₂', 'Ṡ'] 
+              avarohana: ['S', 'N2', 'D2', 'P', 'M2', 'G2', 'R2', 'S']
+59	Dharmavati: 
+              arohana:  ['S', 'R₂', 'G₂', 'M₂', 'P', 'D₂', 'N₃', 'Ṡ']
+              avarohana: ['S', 'N3', 'D2', 'P', 'M2', 'G2', 'R2', 'S']
+60	Neetimati:	
+              arohana: ['S', 'R₂', 'G₂', 'M₂', 'P', 'D₃', 'N₃', 'Ṡ'] 
+              avarohana: ['S', 'N3', 'D3', 'P', 'M2', 'G2', 'R2', 'S']
+61	Kaantaamaṇi:	
+               arohana: ['S', 'R₂', 'G₃', 'M₂', 'P', 'D₁', 'N₁', 'Ṡ'] 
+               avarohana: [ 'S', 'N1', 'D1', 'P', 'M2', 'G3', 'R2', 'S']
+62	Rishabhapriya:	
+               arohana: ['S', 'R₂', 'G₃', 'M₂', 'P', 'D₁', 'N₂', 'Ṡ'] 
+               avarohana: ['S', 'N2', 'D1', 'P', 'M2', 'G3', 'R2', 'S']
+63	Lataangi:	
+               arohana: ['S', 'R₂', 'G₃', 'M₂', 'P', 'D₁', 'N₃', 'Ṡ'] 
+               avarohana: [ 'S', 'N3', 'D1', 'P', 'M2', 'G3', 'R2', 'S']
+64	Vaachaspati:	
+               arohana: ['S', 'R₂', 'G₃', 'M₂', 'P', 'D₂', 'N₂' ,'Ṡ'] 
+               avarohana: [ 'S', 'N2', 'D2', 'P', 'M2', 'G3', 'R2', 'S']
+65	Mechakalyaani:	
+               arohana: ['S', 'R₂', 'G₃', 'M₂', 'P', 'D₂', 'N₃', 'Ṡ'] 
+               avarohana: [ 'S', 'N3', 'D2', 'P', 'M2', 'G3', 'R2', 'S']
+66	Chitraambari:	
+               arohana: ['S', 'R₂', 'G₃', 'M₂', 'P', 'D₃', 'N₃', 'Ṡ']
+               avarohana:  ['S', 'N3', 'D3', 'P', 'M2', 'G3', 'R2', 'S']
+67	Sucharitra:	
+               arohana: ['S', 'R₃', 'G₃', 'M₂', 'P', 'D₁', 'N₁', 'Ṡ'] 
+               avarohana: [ 'S', 'N1', 'D1', 'P', 'M2', 'G3', 'R3', 'S']
+68	Jyoti svarupini:	
+                arohana:  ['S', 'R₃', 'G₃', 'M₂', 'P', 'D₁', 'N₂', 'Ṡ'] 
+                avarohana: ['S', 'N2', 'D1', 'P', 'M2', 'G3', 'R3', 'S']
+69	Dhaathuvardhani:
+           	arohana: ['S', 'R₃', 'G₃', 'M₂', 'P', 'D₁', 'N₃', 'Ṡ'] 
+            avarohana: ['S', 'N3', 'D1', 'P', 'M2', 'G3', 'R3', 'S']
+70	Naasikaabhushaṇi:	
+             arohana: ['S', 'R₃', 'G₃', 'M₂', 'P', 'D₂', 'N₂', 'Ṡ'] 
+             avarohana: ['S', 'N2', 'D2', 'P', 'M2', 'G3', 'R3', 'S']
+71	Kosalam:	
+             arohana: ['S', 'R₃', 'G₃', 'M₂', 'P', 'D₂', 'N₃', 'Ṡ'] 
+             avarohana: [ 'S', 'N3', 'D2', 'P', 'M2', 'G3', 'R3', 'S']
+72	Rasikapriya: 
+             arohana:	['S', 'R₃', 'G₃', 'M₂', 'P', 'D₃', 'N₃', 'Ṡ'] 
+             avarohana: [ 'S', 'N3', 'D3', 'P', 'M2', 'G3', 'R3', 'S']
+
+Janya Ragas 
+
 anandabhairavi:
   arohana: ['S', 'G2', 'R2', 'G2', 'M1', 'P', 'D2', 'P', 'N2', 'S']
   avorahana: ['S', 'N2', 'D2', 'P', 'M1', 'G2', 'R2', 'S']
```

Comment:
please remove this field for each raga. at the moment, each has two avarahanas! It is my mistake for spelling it incorrectly initially